keras/layers/core/input_layer.py

```diff
@@ -17,6 +17,7 @@ def __init__(
         batch_shape=None,
         input_tensor=None,
         name=None,
+        ragged = False,
         **kwargs,
     ):
         # TODO: support for ragged.
```

Comment:
Ragged tensors are actually not supported -- so if there's a documentation discrepancy we should fix the docs, we shouldn't add the argument since there's no actual support for it.

Reply:
Yeah fine then i will make changes in the docs and will undo the changes made in the files.